project_key/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+===========
+Project Key
+===========
+
+This module provides functionality to uniquely identify projects and tasks by simple ``key`` field.
+
+
+Usage
+=====
+
+To use this module functionality you just need to:
+
+On ``project.project`` level:
+
+In Kanban View:
+
+#. Go to Project > Dashboard
+#. Create
+#. Enter project name and use auto generated key or simply override value by entering your own key value.
+
+In Tree View:
+
+#. Go to Project > Configuration > Projects
+#. Create
+#. Enter project name and use auto generated key or simply override value by entering your own key value.
+
+In form View:
+
+#. Go to Project > Dashboard
+#. Open the projects settings
+#. Modify the "key" value
+#. After modifying project key any existing task will be reindexed automatically.
```

Comment:
I find the terminology confusing as this does not concern the sql column index. How about `the code of any existing tasks will be updated automatically` (rename terminology in the code as well).

Reply:
Hi @StefanRijnhart: I've considered your comment and I've actually decided to rename the module to "project_code" and also I've changed the terminology as per your comment

Reply:
As it turns the new code was in collision with module ``project_task_code`` so I switched it back.